embedded/devices/include/mcp23017.h

```diff
@@ -7,15 +7,15 @@
 #define MCP_DIRA_REG 0x00
 #define MCP_DIRB_REG 0x01
 #define MCP_GPIOA_REG 0x12
-#define MCP_GPIOB_REG 0x13
+define MCP_GPIOB_REG 0x13
 
 #define MCP_NUM_PINS 16
 
 #define IS_BANK_A(pin) (pin < (MCP_NUM_PINS >> 1))
 #define GET_REL_PIN(pin) (pin & 0x7)
 
-static const uint8_t rick_directions[16] = {0, 0, 0, 0, 0, 0, 0, 0,
-                                            0, 0, 0, 0, 0, 0, 0, 0};
+    static const uint8_t rick_directions[16] = {0, 0, 0, 0, 0, 0, 0, 0,
+                                                0, 0, 0, 0, 0, 0, 0, 0};
```

Comment:
extra whitespace here as well. looks like the whole declaration got tabbed in one